docs/content/docs/cd/QApps.md

```diff
@@ -9,9 +9,8 @@ type: docs
 
 [QApps Android](http://links.k.avito.ru/qappsAndroid)
 
-Внутреннее хранилище сборок для ручного тестирования.\
-Заливаем в него apk по расписанию из develop.\
-См. конфиг `uploadToQapps` в `build.gradle` приложений.
+This is the internal storage with bulds for manual testing.\
+We publish builds automatically after merge to develop.
 
 ## Как залить вручную в QApps с ветки?
```

Comment:
Should we translate it into English?

Reply:
Yes, as all pages. I just want to move by little steps.
If we require to translate the whole page in case of changes it creates a barrier. I'd like to encourage small translations too.